packages/fxa-content-server/app/scripts/templates/settings/delete_account.mustache

```diff
@@ -11,7 +14,50 @@
     <div class="error"></div>
 
     <form novalidate>
-      <p>{{#t}}If you use this account for Mozilla apps and services like AMO, Pocket, and Screenshots you should first log in and remove any saved personal information. Once you delete your account here you may lose access to it everywhere.{{/t}}</p>
+      <p>{{#t}}Your Firefox Account gives you access to products and services that help keep your web experience secure, private, and productive.{{/t}}</p>
+
+      <p>{{#t}}You currently use:{{/t}}</p>
+
+        <ul class="account-summary-list">
+          {{#subscriptions}}
+            <li>
+              {{productName}}
+            </li>
+          {{/subscriptions}}
+
+          {{#clients}}
```

Comment:
Are any changes needed here to match [@rfk's PR to list all attached clients in the devices and apps view](https://github.com/mozilla/fxa/pull/1262)? From my reading here it looks like this already conforms, 👍, but want to make sure.

Reply:
👍 Thanks for calling this out, I made sure to pull from master yesterday but I know I copied from/referenced that clients view late last week (when I may have been up-to-date anyway) so it doesn't hurt to double check.